plugins/infinispan100/src/main/java/org/radargun/service/InfinispanServerngClustered.java

```diff
@@ -0,0 +1,99 @@
+package org.radargun.service;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import org.radargun.logging.Log;
+import org.radargun.logging.LogFactory;
+import org.radargun.traits.Clustered;
+
+/**
+ * @author Gustavo Lira &lt;glira@redhat.com&gt;
+ */
+public class InfinispanServerngClustered implements Clustered {
```

Comment:
Will it block the current thread execution ?

Reply:
Will work in the same way like is happening currently with `InfinispanServerClustered.java`, I just changed where I'm getting the information, instead of get information from mbeans I'm using Rest v2 API to get cluster information